### DISCUSSION TITLE
Is there anyway to batch download an athlete's activity log?

### DISCUSSION THREAD

**Discussion by ryanppax** [SauceLLC/sauce4strava#127] [General]
I wanted to export data for analysis in wko and would like to get the whole year of data

**Comment by mayfield** [comment]
Hi @ryanppax After you've sync'd an athlete you can use the "Export FIT Files" button from the sync panel...

![image](https://user-images.githubusercontent.com/139316/181838909-aa96f270-8456-4a61-b174-9bc98a77ada1.png)

Here's a video explaining the sync process a bit more..

https://youtu.be/k5JAMvsu39A
